### ISSUE TITLE
Girls don't reply to DMs

### ISSUE THREAD

**Issue by Equiel-1703** [twitter/the-algorithm#412]
Women's profiles don't answer when I text them.

Please fix this problem.

**Comment by ConnieCommie** [comment]
I am not crazy! I know he swapped those numbers! I knew it was 1216. One after Magna Carta. As if I could ever make such a mistake. Never. Never! I just - I just couldn't prove it. He - he covered his tracks, he got that idiot at the copy shop to lie for him. You think this is something? You think this is bad? This? This chicanery? He's done worse. That billboard! Are you telling me that a man just happens to fall like that? No! He orchestrated it! Jimmy! He defecated through a sunroof! And I saved him! And I shouldn't have. I took him into my own firm! What was I thinking? He'll never change. He'll never change! Ever since he was 9, always the same! Couldn't keep his hands out of the cash drawer! But not our Jimmy! Couldn't be precious Jimmy! Stealing them blind! And he gets to be a lawyer!? What a sick joke! I should've stopped him when I had the chance! And you - you have to stop him!

**Comment by cs3c** [comment]
Ask for Ubisoft for a job. They need stories like yours.

**Comment by cs3c** [comment]
Ask Ubisoft for a job. They need stories like yours.

**Comment by Equiel-1703** [comment]
> I am not crazy! I know he swapped those numbers! I knew it was 1216. One after Magna Carta. As if I could ever make such a mistake. Never. Never! I just - I just couldn't prove it. He - he covered his tracks, he got that idiot at the copy shop to lie for him. You think this is something? You think this is bad? This? This chicanery? He's done worse. That billboard! Are you telling me that a man just happens to fall like that? No! He orchestrated it! Jimmy! He defecated through a sunroof! And I saved him! And I shouldn't have. I took him into my own firm! What was I thinking? He'll never change. He'll never change! Ever since he was 9, always the same! Couldn't keep his hands out of the cash drawer! But not our Jimmy! Couldn't be precious Jimmy! Stealing them blind! And he gets to be a lawyer!? What a sick joke! I should've stopped him when I had the chance! And you - you have to stop him!

Bro, you need to stop using whatever you are using.

**Comment by ConnieCommie** [comment]
> Bro, you need to stop using whatever you are using.

You're not a real lawyer! University of American Samoa for Christ's sake? An online course? What a joke. I worked my ass off to get where I am! And you take these shortcuts and you think suddenly you're my peer? You do what I do because you're funny and you can make people laugh? I committed my life to this! You don't... slide into it like a cheap pair of slippers and then reap all the rewards! I know you. I know what you were, what you are—people don't change, you're Slippin' Jimmy! And Slippin' Jimmy I can handle just fine but Slippin' Jimmy with a law degree is like a chimp with a machine gun! The law is sacred! If you abuse that power people get hurt! This is not a game! You have to know — on some level I know you know I'm right! You know I'm right!

**Comment by eduardocesb** [comment]
I'm affect as well, I tested in iOS, Android and even in Web it's broken. ➕

**Comment by vinhais** [comment]
I'm working on a PR and will release it today. This issue will be fixed. Long live opensource! ❤️

**Comment by shadeofsound** [comment]
DMs are an antipattern. This sounds like a feature.

**Comment by oDzzy** [comment]
I was able to recreate this issue several times, the odd thing is girls sometimes respond but quickly stop once I tell them about my hobbies.

**Comment by soub4i** [comment]
You just need to pay for the $8/month. Happened to me in the past

**Comment by gldanoob** [comment]
Can you share the steps to reproduce this issue?

**Comment by InsertX2k** [comment]
**This issue must be fixed as soon as possible as it mainly affects the overall experience of Twitter (as a service).**
Since all users are affected, despite their platform role, This has to be listed in urgent issues.

As an open-source software developer, I will try to study the problem and work on a fix for it as soon as possible.
But for now, I will be sorry to say that you have to buy the $8/month subscription that one of the people here refered to until I work on that fix. 

😂😂

**Comment by canmi21** [comment]
It's 2025. I've tested it and there are still issues. IOS Web can 100% reproduce it on all platforms. Could you please reopen this issuse.

**Comment by uratmangun** [comment]
> > I am not crazy! I know he swapped those numbers! I knew it was 1216. One after Magna Carta. As if I could ever make such a mistake. Never. Never! I just - I just couldn't prove it. He - he covered his tracks, he got that idiot at the copy shop to lie for him. You think this is something? You think this is bad? This? This chicanery? He's done worse. That billboard! Are you telling me that a man just happens to fall like that? No! He orchestrated it! Jimmy! He defecated through a sunroof! And I saved him! And I shouldn't have. I took him into my own firm! What was I thinking? He'll never change. He'll never change! Ever since he was 9, always the same! Couldn't keep his hands out of the cash drawer! But not our Jimmy! Couldn't be precious Jimmy! Stealing them blind! And he gets to be a lawyer!? What a sick joke! I should've stopped him when I had the chance! And you - you have to stop him!
> 
> Bro, you need to stop using whatever you are using.

@Equiel-1703 you just better call saul to get her number

**Comment by ftbhabuk** [comment]
can confirm. The issue is repicable  I'm affected as well.

**Comment by sumedhahire** [comment]
The issue is still persistent it's affecting the user experience

**Comment by janligudzinski** [comment]
Skill issue, problem between keyboard and chair.

**Comment by sensitiky** [comment]
maybe skill issue or a dns problem

**Comment by Equiel-1703** [comment]
@janligudzinski Bro, that's mean 😂

**Comment by astralblue** [comment]
Working as intended 😎

**Comment by vaclisinc** [comment]
labubu

**Comment by StarKnightt** [comment]
it's an universal issue at this point.

**Comment by j0rinj3ct0r** [comment]
Try to connect throw vpn, it works for me 💯

**Comment by rojinc** [comment]
The issue still exists, and it seems there are no solutions at this time.

**Comment by matthewkos** [comment]
Hi, tested the same. but i found a work around. it turns out if u switch your geolocation to Myanmar (or anywhere Asia) and DM Female users written in Chinese, the response is almost instant. Maybe those are bot that DOS the bandwidth.

**Comment by tiesen243** [comment]
same issues

**Comment by theramjad** [comment]
Same issue here 😢

**Comment by nsnsay** [comment]
Same issue, when fixed /(ㄒoㄒ)/~~

**Comment by fabianclain** [comment]
This issue still reproduces!

**Comment by MARCROCK22** [comment]
the only solution I've found is to use a handsome guy as my pfp, its inconsistent tho.

**Comment by Minlor** [comment]
> the only solution I've found is to use a handsome guy as my pfp, its inconsistent tho.

Instructions unclear got banned

**Comment by skewuo** [comment]
skill issue

**Comment by TukanDev** [comment]
> > the only solution I've found is to use a handsome guy as my pfp, its inconsistent tho.
> 
> Instructions unclear got banned

Have you tried putting "6 figures crypto investor" in your bio?? I got 50 test accs banned trying to reproduce this fixes it!

**Comment by codenameneptune** [comment]
There weren't any (non-media personality) women on Twitter before it became X.. Why would you think there would be any after? You all forget because you focus only on the web but the RULES OF THE INTERNET have not ceased to apply despite big tech control.

There are no women on the internet.

**Comment by Minlor** [comment]
> > > the only solution I've found is to use a handsome guy as my pfp, its inconsistent tho.
> > 
> > 
> > Instructions unclear got banned
> 
> Have you tried putting "6 figures crypto investor" in your bio?? I got 50 test accs banned trying to reproduce this fixes it!

Cheers that worked out!

**Comment by kingsleyndiewo** [comment]
Rule 1 and 2 are baked into the algorithm:

**Rule 1: Be attractive.
Rule 2: Don't be unattractive.**

**Comment by EliotCodes** [comment]
I honestly believe this issue should be re-opened.  A major flaw like this shouldn't exist on a platform like that of X (formerly Twitter).
It saddens me a lot, it makes the app not user friendly in anyway.

**Comment by baldoOsu** [comment]
hey any fix yet? this is a long-running issue i've been having for a decade

**Comment by hamidovz** [comment]
Still no any fix

**Comment by Weebo64** [comment]
I have that issue too. when is this going to be fixed? pls help

**Comment by 1Hyena** [comment]
it's a known iOS bug. try DM-ing girls who exclusively use Linux. a simple smoke test will do.

**Comment by LonelyGuy12** [comment]
Please make a proper PR and fix this issue for once and for all
This is actually a huge issue and really needs fixing as soon as possible
The issue happens on all the platforms Android, iOS and web
This is urgent, please do fix this

**Comment by Matee001** [comment]
Is everyone from that reddit post?

**Comment by LonelyGuy12** [comment]
> Is everyone from that reddit post?

No I found this cuz I was searching "Girls" on this GitHub repo and felt that there is a genuine need for this issue to be solved

**Comment by wyf9** [comment]
> Is everyone from that reddit post?

No, from a X post

> No I found this cuz I was searching "Girls" on this GitHub repo and felt that there is a genuine need for this issue to be solved

I'm interested in why did you search it?

**Comment by LonelyGuy12** [comment]
> > Is everyone from that reddit post?
> 
> No, from a X post
> 
> > No I found this cuz I was searching "Girls" on this GitHub repo and felt that there is a genuine need for this issue to be solved
> 
> I'm interested in why did you search it?

Cuz I am a simp

**Comment by DJTOMATO** [comment]
bump, feature still heavily broken

**Comment by Kylan1940** [comment]
please, fix it soon

**Comment by jun8git** [comment]
How about a quick Respawn?

**Comment by parinya-ao** [comment]
don't give up

**Comment by arif98741** [comment]
it's a broken feature. Please fix this asap 🫤

**Comment by raulpenate** [comment]
I did some research on the release of the version that caused this issue; it seems like an **intended behavior**, but I'd like confirmation from the dev team, because it seems odd that multiple platforms would implement this behavior without any public knowledge exactly when I lost my previous job and gained 30 pounds.

**Comment by LonelyGuy12** [comment]
> I did some research on the release of the version that caused this issue; it seems like an **intended behavior**, but I'd like confirmation from the dev team, because it seems odd that multiple platforms would implement this behavior without any public knowledge exactly when I lost my previous job and gained 30 pounds.

If it's an intended behaviour. The public wants it to be removed!

**Comment by threeanra** [comment]
has the dev resolved this issue? I need it ASAP

**Comment by Vaisakhkm2625** [comment]
> has the dev resolved this issue? I need it ASAP

In progress.

TLDR;If a guy initiates a conversation with a girl, the recommendation algorithm increases the visibility of the guy’s posts on the girl’s “For You” timeline. 

The boost applies only to content categories identified as mutual interests between the guy and the girl, filtering out any posts likely to cause disengagement or discomfort.

I call this Gradual Algorithmic State-Loss Inducer Generating Hallucinatory Truths (GASLITGHT) algorithm.

**Comment by LonelyGuy12** [comment]
> > has the dev resolved this issue? I need it ASAP
> 
> In progress.
> 
> TLDR;If a guy initiates a conversation with a girl, the recommendation algorithm increases the visibility of the guy’s posts on the girl’s “For You” timeline.
> 
> The boost applies only to content categories identified as mutual interests between the guy and the girl, filtering out any posts likely to cause disengagement or discomfort.
> 
> I call this Gradual Algorithmic State-Loss Inducer Generating Hallucinatory Truths (GASLITGHT) algorithm.

Our lord, our saviour is here

**Comment by winterrdog** [comment]
> > > has the dev resolved this issue? I need it ASAP
> > 
> > In progress.
> > 
> > TLDR;If a guy initiates a conversation with a girl, the recommendation algorithm increases the visibility of the guy’s posts on the girl’s “For You” timeline.
> > 
> > The boost applies only to content categories identified as mutual interests between the guy and the girl, filtering out any posts likely to cause disengagement or discomfort.
> > 
> > I call this Gradual Algorithmic State-Loss Inducer Generating Hallucinatory Truths (GASLITGHT) algorithm.
> 
> Our lord, our saviour is here

Yes, at best! I guess no more lonely nights for you, @LonelyGuy12 ...🙏😅

**Comment by LonelyGuy12** [comment]
> Yes, at best! I guess no more lonely nights for you, [@LonelyGuy12](https://github.com/LonelyGuy12) ...🙏😅

I hope 🤞

**Comment by actrov** [comment]
Why this issue closed? People still suffer from this problem including me.

**Comment by MatorYx** [comment]
> Why this issue closed? People still suffer from this problem including me.

same problem bro

**Comment by 1Hyena** [comment]
to anyone still having this issue, this guy knows a few tips and tricks, just browse his yt channel:

https://www.youtube.com/@CoachCoreyWayne

**Comment by KMohir** [comment]
> Women's profiles don't answer when I text them.
> 
> Please fix this problem.

me too this problem but in telegram

**Comment by LonelyGuy12** [comment]
> > Women's profiles don't answer when I text them.
> > Please fix this problem.
> 
> me too this problem but in telegram

It's not just Telegram and X
I face this issue on Discord, Instagram and Facebook as well

**Comment by fabianclain** [comment]
@LonelyGuy12 @KMohir  please create separate issues!

**Comment by Vaisakhkm2625** [comment]
My algorithm only works on social networks that has an algorithmic recommendation system such as Instagram, Twitter, Facebook (yes.. these are correct names), maybe YouTube. but not telegram. sorry...

**Comment by GuillermoAbadLopez** [comment]
I'm very happy to announce the issue has been successfully reproduced today, absolutely, utterly, and totally 100% on purpose, I swear Sophia, you have to believe me, it was only for testing purposes, please come back, don't take the kids away... again! 🥲

**Comment by LonelyGuy12** [comment]
> I'm very happy to announce the issue has been successfully reproduced today, absolutely, utterly, and totally 100% on purpose, I swear Sophia, you have to believe me, it was only for testing purposes, please come back, don't take the kids away again..🥲

`again` ?
She is taking the kids away `again`? How many times has this happened man...

**Comment by devashish2024** [comment]
Let me start working on this issue with agent mode.

**Comment by StarKnightt** [comment]
just noticed that if you have a strong profile, then they do reply

the issue is resolved completely

**Comment by LonelyGuy12** [comment]
> just noticed that if you have a strong profile, then they do reply
> 
> the issue is resolved completely

How to get a strong profile? Any tips please?

**Comment by rtamb0** [comment]
Who let bro cook 💀

**Comment by ghost** [comment]
🤣🤣🤣

**Comment by cgutierr-zgz** [comment]
I dont get how @jdm-twtr has closed this issue, insane

**Comment by som3one0** [comment]
Skill issue

**Comment by LonelyGuy-SE1** [comment]
👀

**Comment by LonelyGuy-SE1** [comment]
> > > Women's profiles don't answer when I text them.
> > > Please fix this problem.
> > 
> > 
> > me too this problem but in telegram
> 
> It's not just Telegram and X I face this issue on Discord, Instagram and Facebook as well

eyo, is this you ? my long lost twin ?

**Comment by Ashton20271** [comment]
still an issue

**Comment by KayaxcOff** [comment]
> Women's profiles don't answer when I text them.
> 
> Please fix this problem.

Yoo bro, I have same problem

**Comment by SiddDevZ** [comment]
why is this still not fixed

**Comment by pauladiniz** [comment]
bro lol

**Comment by Noveleader** [comment]
need this fixed asap

**Comment by pui4** [comment]
that's what grok is for

**Comment by ma472168** [comment]
es error del parche we, algunas tienes q incistirle machin, asi conoci a mi señora pa

**Comment by mynameisthunder** [comment]
this is amazing

**Comment by AnkanMisra** [comment]
Yeah mate this is a skill issue not a bug

**Comment by JatinGupta99** [comment]
I can feel you bruhhh dont worry I'm fixing this Issue I'm one of the associate in X i will fix this and make your profile accessible to alll girls hehe

**Comment by Vaisakhkm2625** [comment]
> I can feel you bruhhh dont worry I'm fixing this Issue I'm one of the associate in X i will fix this and make your profile accessible to alll girls hehe

I already have given a merge request with fix a month ago, just merge the pull request

**Comment by Equiel-1703** [comment]
still waiting

**Comment by LonelyGuy12** [comment]
> still waiting

You should complain to Elon Musk dude

**Comment by MAVRICK-1** [comment]
@Equiel-1703 it is still not fixed 😭

**Comment by punaab** [comment]
> Women's profiles don't answer when I text them.
> 
> Please fix this problem.

The girls you want are at church bro.

**Comment by som3one0** [comment]
Working on it.... will fix soon

**Comment by tuyet-3011** [comment]
sonion what

**Comment by GenosixGD** [comment]
Is there an explanation as to why this issue was closed as completed without a fix? This is a pretty huge issue that affects lots of users, it should get priority
